8p/EightPointsGuzzleBundle
# Remove jQuery, optimize JS

| Q                | A
| ---------------- | -----
| Bug fix          | yes
| New feature      | no
| BC breaks        | no
| Deprecations     | no
| Tests pass       | yes
| Fixed tickets    | [#188]
| License          | MIT

What I did:
- remove jQuery - I don't think we need the whole library for 1-2 methods that can be written on clean JS + I think issue in #188 was cause because conflict between different jQuery versions. It's better to remove it at all than to fix conflicts.
- moved script tab out of the loop - there was script tab in look and bloc was generated for each request. Now there are only one script tag and js code per whole page
- removed id if there are multiple elements with same id. It's not allowed in html to have multiple elements with same id. Technically it can work, but its better to avoid such as cases.
- remove variable that tracks first element in look. Used `loop.index0` for that

@florianpreusner what do you think? It's not finished and tested good yet but I want to know your opinion. Finally we can move back jQuery anytime if we will observe that some functionality that strictly depends on it appears.
